### PR TITLE
Display, Download, Delete Uploaded documents (DEV-88)

### PR DESCRIPTION
**Pull request by Davit-BetterAngels**
DEV-88

This PR does not include View and Edit

### CODE REVIEW

File: libs/expo/betterangels/src/lib/screens/Client/Docs/Documents.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { View } from 'react-native';
+import { ClientDocumentType, Maybe } from '../../../apollo';
+import { DocumentModal } from '../../../ui-components';
+
+interface IDocumentsProps {
+  expanded: undefined | string | null;
+  setExpanded: (expanded: undefined | string | null) => void;
+  data: ClientDocumentType[];
+  clientId: string;
+  title: 'Other' | 'Doc-Ready' | 'Consent Form';
```
Comment (mikefeldberg) [inline libs/expo/betterangels/src/lib/screens/Client/Docs/Documents.tsx:14]: I think the three sections are: doc-ready, forms, and other. 
```suggestion
  title: 'Other' | 'Doc-Ready' | 'Forms';
```

File: libs/expo/betterangels/src/lib/screens/Client/Docs/index.tsx [hunk 2]
```diff
@@ -39,6 +48,43 @@ export default function Docs({
...
+            <Documents
+              title="Doc-Ready"
+              {...props}
+              data={
+                client?.clientProfile.docReadyDocuments as ClientDocumentType[]
+              }
+              clientId={client?.clientProfile.id}
+            />
+          )}
+        {client?.clientProfile.consentFormDocuments &&
```
Comment (mikefeldberg) [inline libs/expo/betterangels/src/lib/screens/Client/Docs/index.tsx:63]: `consentFormDocuments` should be `formDocuments` everywhere
```suggestion
        {client?.clientProfile.formDocuments &&
```

**Review by mikefeldberg — APPROVED** [review]
looks great! just need to update "consent forms" to "forms" everywhere

File: apps/betterangels/app.config.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ export default {
     name: IS_PRODUCTION ? 'BetterAngels' : 'BetterAngels (Dev)',
     slug: 'betterangels',
     scheme: IS_PRODUCTION ? 'betterangels' : 'betterangels-dev',
-    version: '1.0.20',
+    version: '1.0.21',
```
Comment (shiozakiyuko) [inline apps/betterangels/app.config.js:16]: Hi @Davit-BetterAngels 

I noticed a couple things:

1. I created a new profile, and uploaded some files. What I noticed was the date: it shows 11/09/23 on all of the files. 
Can you change this to whenever the date the user uploaded? 
2. When I tap the Download the file button, it takes about 4-5 seconds to see the menu to select the destination. Is it possible to make it faster? 
3. The folder name should be "Forms" but I see "Consent Form". Let me know I should create a ticket or this is the right PR to mention this but in the DEV-88, I see it mentioned "Forms". Can you modify this?

Everything else looks good on Download / Delete (I see some of the things that need to be fixed but I will create tickets for them.) Thanks so much!

File: libs/expo/betterangels/src/lib/screens/Client/Docs/index.tsx (hunk 2 above)
Comment (Davit-BetterAngels) [reply libs/expo/betterangels/src/lib/screens/Client/Docs/index.tsx:63]: not availabe on backend. lets discuss this tomorrow

Comment (mikefeldberg) [reply libs/expo/betterangels/src/lib/screens/Client/Docs/index.tsx:63]: oh, my bad. I can update this

**Pull request merged**